### PR TITLE
add key feautures roadmap for v1.6

### PR DESCRIPTION
**Pull request by Thor-wl**
Signed-off-by: Thor-wl <13164644535@163.com>

### CODE REVIEW

File: docs/community/roadmap.md [hunk 1]
```diff
@@ -151,10 +151,29 @@ Investigate to cooperate with argo to support job dependencies.
...
+### Support configuration hot update(Delay)
 * Description: Add hot update for Volcano components arguments.
 * Issue: https://github.com/volcano-sh/volcano/issues/1326
 
+## v1.6
+### Support Dynamic Scheduling Based on Realtime Load
+* Description: Support dynamic scheduling based on realtime load.
+* Priority: High
+* Issue: https://github.com/volcano-sh/volcano/issues/1777
+* Owner: @william-wang
```
Comment (k82cn) [inline docs/community/roadmap.md:163]: let's add a shepherd for each features, the shepherd should be one of approvor :)

**Comment by Thor-wl** [comment]
Are there any suggestions? @william-wang

**Comment by Thor-wl** [comment]
Are there any suggestions? @william-wang

**Comment by Yikun** [comment]
one suggestion: give a certain time for each history verison, just like v1.0, v1.2, v1.3 did, give a roughly time for future version.

File: docs/community/roadmap.md (hunk 1 above)
Comment (Thor-wl) [reply docs/community/roadmap.md:163]: > let's add a shepherd for each features, the shepherd should be one of approvor :)

OK. Let me add one for the last feature.

**Comment by Thor-wl** [comment]
> one suggestion: give a certain time for each history verison, just like v1.0, v1.2, v1.3 did, give a roughly time for future version.

Thanks for your advice. Normally, the community will release a min release around **every 3 monthes**. Bug fix releases are launched on demand. But I still think your advice is meaningful. Let me add for that.

**Comment by Yikun** [comment]
Thanks. This is important for the upstream community to understand our release cadence.

FYI:

[1] https://kubernetes.io/releases/release/
[2] https://spark.apache.org/versioning-policy.html

File: docs/community/roadmap.md [hunk 2]
```diff
@@ -95,7 +95,7 @@ Investigate to cooperate with argo to support job dependencies.
 * Issue: https://github.com/volcano-sh/devices/issues/12
 * Owner: @peiniliu
 
-## v1.4 (To be updated)
+## v1.4
```
Comment (Yikun) [inline docs/community/roadmap.md:98]: When a release published, we'd also better history publish date for version.

File: docs/community/roadmap.md [hunk 3]
```diff
@@ -126,20 +126,20 @@ Investigate to cooperate with argo to support job dependencies.
 * Issue: https://github.com/volcano-sh/volcano/issues/1284
 * Owner: @rudeigerc
 
-## v1.5 (To be updated)
-### Support Hierarchy Queue
+## v1.5
```
Comment (Yikun) [inline docs/community/roadmap.md:129]: same

File: docs/community/roadmap.md [hunk 4]
```diff
@@ -151,10 +151,29 @@ Investigate to cooperate with argo to support job dependencies.
 * Issue: https://github.com/volcano-sh/volcano/issues/1627
 * Owner: @hwdef @shinytang6 @Thor-wl
 
-### Support configuration hot update
+### Support configuration hot update(Delay)
 * Description: Add hot update for Volcano components arguments.
 * Issue: https://github.com/volcano-sh/volcano/issues/1326
 
+## v1.6(To Be Released around May 15, 2022)
```
Comment (Yikun) [inline docs/community/roadmap.md:158]: Good doc

File: docs/community/roadmap.md (hunk 1 above)
Comment (Yikun) [reply docs/community/roadmap.md:163]: Yep, this is a good idea.

File: docs/community/roadmap.md (hunk 3 above)
Comment (Thor-wl) [reply docs/community/roadmap.md]: Resolved

File: docs/community/roadmap.md (hunk 2 above)
Comment (Thor-wl) [reply docs/community/roadmap.md]: Resolved

**Comment by Thor-wl** [comment]
> Thanks. This is important for the upstream community to understand our release cadence.

Got it. Thanks for the report. That make senses.

**Review by wpeng102 — APPROVED** [review]
/lgtm

**Pull request merged**